federation/pkg/fedtypes/secret.go

```diff
@@ -27,20 +28,25 @@ import (
 	kubeclientset "k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
 )
 
+const (
+	SecretKind           = "secret"
```

Comment:
better to use camel case for SecretKind, it is the convention used for kube object kind.